__docs__/wonder-blocks-button/button.argtypes.ts

```diff
@@ -7,8 +7,22 @@ export default {
         description: "Text to appear on the button.",
         type: {name: "string", required: true},
     },
-    icon: {
-        description: "A Phosphor icon asset (imported as a static SVG file).",
+    startIcon: {
+        description: `A Phosphor icon asset (imported as a static SVG file)
+            that will appear at the start of the button (left for LTR, right
+            for RTL).`,
+        type: {name: "other", value: "PhosphorIconAsset", required: false},
+        control: {type: "select"},
+        options: IconMappings,
```

Comment:
**suggestion:** This is something that I missed when I changed this. Turns out that we need to use both keys to preserve the icon selection in a separate Storybook tab.

```suggestion
        options: Object.keys(IconMappings),
        mapping: IconMappings,
```


e.g. https://khan.github.io/wonder-blocks/?path=/story/button--default&args=color:destructive

If you change the props in the `Controls` tab, the icon is not currently preserved, but if we make this change, then it's added to the query string so it can be included in a "permalink" version.

Permalink version: https://khan.github.io/wonder-blocks/iframe.html?args=color:destructive&id=button--default&viewMode=story